lib/client/vm_api.rb

```diff
@@ -6,7 +6,11 @@ def vm(vm_id, opts={})
     end
 
     def vms(opts={})
-      headers = {:accept => "application/xml; detail=disks; detail=nics; detail=hosts"}
+      if opts[:without_details]
+        headers = {:accept => "application/xml"}
+      else
+        headers = {:accept => "application/xml; detail=disks; detail=nics; detail=hosts"}
+      end
       path = "/vms"
       path += search_url(opts) unless filtered_api
```

Comment:
@oourfali <https://github.com/oourfali> this can happen when you use
different compute resources for quota, or just for different datacenter /
cluster.

so IMHO, its a valid usage case to use the filter api - granted it would be
nicer if there was no such limitation in ovirt api :)